### PR TITLE
Add lgamma_op kernel and unittest

### PR DESCRIPTION
**Pull request by levi131**
### PR types
New features

### PR changes
OPs

### Describe
Add lgamma op and unit test.

### CODE REVIEW

File: paddle/fluid/operators/lgamma_op.cc [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+class LgammaOpMaker : public framework::OpProtoAndCheckerMaker {
+ public:
+  void Make() override {
+    AddInput("X", "(Tensor), The input tensor of lgamma op.");
+    AddOutput("Out", "(Tensor), The output tensor of lgamma op.");
+    AddComment(R"DOC(
+Lgamma Operator.
+
+This operator is used to perform elementwise lgamma for input $X$.
```
Comment (zhhsplendid) [inline paddle/fluid/operators/lgamma_op.cc]: Tiny English issue:

"This operator performs ..." is easier

Comment (levi131) [reply paddle/fluid/operators/lgamma_op.cc]: modified

**Review by Aurelius84 — APPROVED** [review]
LGTM

**Review by Xreki — APPROVED** [review]
LGTM for op benchmark ci

**Review by juncaipeng — APPROVED** [review]
LGTM

**Pull request merged**